### ISSUE TITLE
TTRSS update broke the Plugin again

### ISSUE THREAD

**Issue by Nepochal** [joshp23/ttrss-to-wallabag-v2#30]
Today I updated TTRSS to the current master-state (commit 70a9f301). When I try to use your plugin I get the following error:

```
ReferenceError: notify_progress is not defined
    at postArticleToWallabag (https://my.domain/index.php:92:50)
    at HTMLImageElement.onclick (https://my.domain/index.php#f=20&c=0:1:1)
```

Please tell me, if you need more information. Thank you very much.

**Comment by joshp23** [comment]
can you tell me what the latest commit was for your installation of ttrss? there has been a lot of activity lately.

**Comment by Nepochal** [comment]
Of course. I tried the latest commit [c4b55d2eafce8965cc8852f5e9c37201380cb3e2](https://git.tt-rss.org/fox/tt-rss/commit/c4b55d2eafce8965cc8852f5e9c37201380cb3e2) a few moment ago.

**Comment by joshp23** [comment]
I was able to reproduce the error.
The console seems to produce some meaningful errors, each which follow
`Loading failed for the <script> with source “https://ttrss.example.com/lib/scriptaculous/controls.js”.`
After that error, the next is that the wallabag js fails to load, etc.

After removing the `notify_progress` line, it complains that `param_escape` is not defined, and so on.

**Comment by joshp23** [comment]
Please see [this](https://discourse.tt-rss.org/t/scriptaliculous-controls-js-failing-to-load-after-recent-updates-breaking-plugins/1741) thread on the ttrss forums.

**Comment by joshp23** [comment]
Fixed with f586edb and is in the latest release (1.7.0)
